EMacco/abarian-backend
# #170469260 payment method migration

#### What does this PR do?
This PR includes payment method in orders migration

#### Description of Task to be completed?
- update the orders migration to include payment method

#### How should this be manually tested?
- checkout to this branch
- run the command `yarn run refreshdb`
- payment method should be included as one of the table fields

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#170469260](https://www.pivotaltracker.com/story/show/170469260)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
